### PR TITLE
Policy: Allow backport of independent features

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:
Clarify that we allow backporting features which do not rely on external dependencies.

#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,14 +19,20 @@
 
 CRI-O follows the Kubernetes release cycles with respect to its minor versions
 (`1.x.y`). Patch releases (`1.x.z`) for CRI-O are not in sync with those from
-Kubernetes, because those are scheduled for each month, whereas CRI-O provides
+Kubernetes, because they are scheduled for each month, whereas CRI-O provides
```
Comment (haircommander) [inline README.md:22]: hah same note as in cri-tools https://github.com/kubernetes-sigs/cri-tools/pull/1246/files#r1300154003

Comment (haircommander) [reply README.md:22]: I'm fine as a follow up too

Comment (saschagrunert) [reply README.md]: Yep, fixed as suggested.

**Comment by saschagrunert** [comment]
@cri-o/cri-o-maintainers PTAL

File: README.md [hunk 2]
```diff
@@ -18,15 +18,21 @@
...
 This means that CRI-O also follows the Kubernetes `n-2` release version skew
 policy when it comes to feature graduation, deprecation or removal. This also
-applies to features which are independent from Kubernetes.
+applies to features which are independent from Kubernetes. Nevertheless, feature
+backports to supported release branches, which are independent from Kubernetes
+or other tools like cri-tools, are still possible. This allows CRI-O to decouple
+from the Kubernetes release cycle and have enough flexibility when it comes to
+implement new features. Every feature to be backported will be a case by case
+decision of the community while the overall compatibility matrix should not be
+compromised.
```
Comment (haircommander) [inline README.md:35]: one note: how should we handle configuration changes? we've historically used minor version bumps to make changes to the config, but there could be cases where we'd want to backport config changes (and have done so in the past some)

Comment (saschagrunert) [reply README.md:35]: I'd say we allow backporting non-breaking config changes.

**Pull request merged**